app/assets/src/components/utils/links.js

```diff
@@ -43,10 +43,31 @@ const openUrlInPopupWindow = (url, windowName, windowWidth, windowHeight) => {
   );
 };
 
+const postToUrlWithCSRF = (url, params) => {
```

Comment:
this function might be better in `src/api/core.js`

Reply:
I took a look on `core.js`, and it seems the functions over there are all used to invoke json endpoints, whereas `links.js` has functions that replace the current location (which is the case here, with the difference I'm using a post operation to do that).
That said, it looked to me to be more logical to put this method on `links.js`. What do you think?

Reply:
I see, `links.js` sounds good then